client/app/src/main/java/com/example/criengine/Fragments/RequestedBooksFragment.java

```diff
@@ -68,17 +78,46 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                     @Override
                     public void onSuccess(List<Book> books) {
                         borrowerBooks.addAll(books);
-                        bookNameTextView.setAdapter(borrowerBooksListAdapter);
-//                        borrowerBooksListAdapter.notifyDataSetChanged();
+                        displayBooks.addAll(books);
+                        borrowerBooksListAdapter.notifyDataSetChanged();
                     }
                 }
         );
 
         bookNameTextView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
             @Override
             public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
-                // TODO: Navigate to book-view (non owner view)
+                Intent intent = new Intent(view.getContext(), MyBookActivity.class);
```

Comment:
we need to navigate to non owner view; my book activity is the owner view (hence the "my")

Reply:
I don't believe we have the view book activity for non-owners so you're gonna want to bring back that TODO @ducklin5